controllers/src/userController/main.cpp

```diff
@@ -14,11 +15,15 @@ void setup()
   Wire.begin();
 
   AppRadio::init();
+
+  Lcd::init();
 }
 
 void loop()
 {
   AppRadio::radio.checkRDS();
   Controllers::readAndProcess();
+  Controllers::displayFreq();
```

Comment:
I suggest refreshing values on display only on changes. Implement function like this in AppRadio methods, next to Serial.printfs

this gives uC more time for other things like reading RDS

Reply:
Following this, I think it's worth it to move all display functions to LCD class

Reply:
But if procedure require e.g two lines I think it shouldn't be moved to separated function